TS2309: An export assignment cannot be used in a module with other exported elements.

Since v1.4.1, TypeScript usage with in a module with `import` keyword is having TS2309 error.

**package.json**
```json
{
  "private": true,
  "dependencies": {
    "it-pushable": "^1.4.1",
    "typescript": "^4.1.3"
  }
}
```

**tsconfig.json**
```jsonc
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "moduleResolution": "node",
    "strict": true,
    "target": "ES2019"
  },
  "include": [
    "script.ts"
  ],
  "exclude": [
    "**/node_modules"
  ]
}
```

**script.ts**
```ts
import pushable from "it-pushable";

pushable();
```

**Compile command**
```bash
./node_modules/.bin/tsc
```

**Error message**
```text
node_modules/it-pushable/index.d.ts:24:1 - error TS2309: An export assignment cannot be used in a module with other exported elements.

24 export = pushable
   ~~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,24 +1,26 @@
-export interface Pushable<T> extends AsyncIterable<T> {
-  push: (value: T) => this,
-  end: (err?: Error) => this
+declare namespace pushable {
+  export interface Pushable<T> extends AsyncIterable<T> {
+    push: (value: T) => this,
+    end: (err?: Error) => this
+  }
+
+  export interface PushableV<T> extends AsyncIterable<T[]> {
+    push: (value: T) => this,
+    end: (err?: Error) => this
+  }
+
+  type Options = {
+    onEnd?: (err?: Error) => void,
+    writev?: false
+  }
+
+  type OptionsV = {
+    onEnd?: (err?: Error) => void,
+    writev: true
+  }
 }
 
-export interface PushableV<T> extends AsyncIterable<T[]> {
-  push: (value: T) => this,
-  end: (err?: Error) => this
-}
-
-type Options = {
-  onEnd?: (err?: Error) => void,
-  writev?: false
-}
-
-type OptionsV = {
-  onEnd?: (err?: Error) => void,
-  writev: true
-}
-
-declare function pushable<T> (options?: Options): Pushable<T>
-declare function pushable<T> (options: OptionsV): PushableV<T>
+declare function pushable<T> (options?: pushable.Options): pushable.Pushable<T>
+declare function pushable<T> (options: pushable.OptionsV): pushable.PushableV<T>
 
 export = pushable

